deploy/Makefile

```diff
@@ -1,21 +1,50 @@
 ENV_FILE=.env
 include $(ENV_FILE)
 
-DOCKER_REMOTE=ssh://$(ZEN_WEB_REMOTE)
-PROJECT_NAME=harmony
+DOCKER_HOST ?= ssh://$(ZEN_WEB_REMOTE)
```

Comment:
For localhost - one would want to use unix:///var/run/docker.sock - so only suck in ZEN_WEB_REMOTE if DOCKER_HOST is not specified.